index.html

```diff
@@ -33,9 +33,149 @@
+            <label>Javascript</label>
+            <label>CSS</label>
+            <label>HTML</label>
+           </div>
+           <button class="see-post"><p class="see-button">see project</p></button>
+        </div>
+        <div class="grid-item fiveth"><div class="item image"></div> 
+        <h2 class="title">Multi-Post Stories Gain + Glory</h2>
+        <div class="tagcontainer">
+         <label>Ruby on Rails</label>
+         <label>Javascript</label>
+         <label>CSS</label>
+         <label>HTML</label>
+        </div>
+        <button class="see-post"><p class="see-button">see project</p></button></div>
+        <div class="grid-item sixth">
+            <div class="item image"></div> 
+           <h2 class="title">Multi-Post Stories Gain + Glory</h2>
+           <div class="tagcontainer">
+            <label>Ruby on Rails</label>
+            <label>Javascript</label>
+            <label>CSS</label>
+            <label>HTML</label>
+           </div>
+           <button class="see-post"><p class="see-button">see project</p></button>
+        </div>
+        <div class="grid-item ">
+            <div class="item image"></div> 
+           <h2 class="title">Multi-Post Stories Gain + Glory</h2>
+           <div class="tagcontainer">
+            <label>Ruby on Rails</label>
+            <label>Javascript</label>
+            <label>CSS</label>
+            <label>HTML</label>
+           </div>
+           <button class="see-post"><p class="see-button">see project</p></button>
+        </div>
+    </div>
+</section>
+<section>
+    <div class="grid-container-another">
+        <div class="about-me">
+            <h2 class="about">About Me</h2>
+            <p class="about-myself"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hesitate to contact me.</p>
+                <button class="get-my-resume"><p class="resume">Get My Resume</p></button>
+        </div>
+        <div class="grid-item-another">
+            <img class="skills languagelogo" src="./icon-languages@2x.png" alt="language logo">
+            <h3 class="language">Languages</h3>
+            <div class="mini-flex-container">
+                <label class="cli mini-item">Javascript</label>
+                 <label class=" mini-item">HTML</label>
+                 <label class=" mini-item">Ruby</label> 
+                 <label class=" mini-item">CSS</label>  
+             </div>
+        </div>
+        <div class="grid-item-another">
+            <img class="skills framelogo" src="./icon-frameworks@2x.png" alt="framework logo">
+            <h3 class="language">Frame Work</h3>
+            <div class="mini-flex-container">
+                <label class="cli mini-item">React.js</label>
+                 <label class=" mini-item">Rspec</label>
+                 <label class=" mini-item">capybara</label>  
+             </div>
```

Comment:
- [ ] Please fix the design for each cards in terms of their fonts, color and spacing between the technologies they are inconsistent and not like the below figma template :+1: 
![card-about](https://user-images.githubusercontent.com/89055502/235299888-1ace9934-9f27-4967-8858-7d0fb5a32e71.png)